src/components/theme/Footer/Footer.jsx

```diff
@@ -82,42 +94,23 @@ const Footer = ({ intl }) => {
           />
         </Segment>
         <List horizontal inverted>
-          {/* wrap in div for a11y reasons: listitem role cannot be on the <a> element directly */}
-          <div role="listitem" className="item">
-            <Link
-              className="item"
-              to={settings.isMultilingual ? `/${lang}/sitemap` : '/sitemap'}
-            >
-              <FormattedMessage id="Site Map" defaultMessage="Site Map" />
-            </Link>
-          </div>
-          <div role="listitem" className="item">
-            <Link
-              className="item"
-              to={
-                settings.isMultilingual
-                  ? `/${lang}/accesibility-info`
-                  : '/accesibility-info'
-              }
-            >
-              <FormattedMessage
-                id="Accessibility"
-                defaultMessage="Accessibility"
-              />
-            </Link>
-          </div>
-          <div role="listitem" className="item">
-            <Link
-              className="item"
-              to={
-                settings.isMultilingual
-                  ? `/${lang}/contact-form`
-                  : '/contact-form'
-              }
-            >
-              <FormattedMessage id="Contact" defaultMessage="Contact" />
-            </Link>
-          </div>
+          {siteActions?.map((item) => (
+            <div role="listitem" className="item">
+              <Link
+                className="item"
+                to={
+                  settings.isMultilingual
+                    ? `/${lang}/${flattenToAppURL(item.url)}`
+                    : flattenToAppURL(item.url)
```

Comment:
Depending on the setup, the urls from the backend will have the /api/ (or other paths signaling the Plone object ID) or not, does this `flattenToAppURL` handle it?

Reply:
yes, As far as the API or internalApiPath is set, it does take care of that: https://github.com/plone/volto/blob/master/src/helpers/Url/Url.js#L100